dali/operators/sequence/optical_flow/optical_flow_impl/optical_flow_impl.cc

```diff
@@ -14,28 +14,48 @@
 
 #include <dlfcn.h>
 
-#include "dali/operators/sequence/optical_flow/turing_of/optical_flow_turing.h"
+#include "dali/operators/sequence/optical_flow/optical_flow_impl/optical_flow_impl.h"
 #include "dali/core/device_guard.h"
 
 namespace dali {
 namespace optical_flow {
 
 namespace {
 
-void VerifySupport(NV_OF_STATUS status) {
+NV_OF_STATUS VerifySupport(NV_OF_STATUS status) {
   switch (status) {
-    case NV_OF_SUCCESS:
-      return;
     case NV_OF_ERR_OF_NOT_AVAILABLE:
     case NV_OF_ERR_UNSUPPORTED_DEVICE:
       throw unsupported_exception("Feature unsupported");
     default:
-      DALI_FAIL("Optical flow failed, code: " + std::to_string(status));
+      return status;
   }
 }
 
 }  // namespace
 
+void ConvertGridToSize(VectorGridSize grid_size, size_t width, size_t height,
+                       size_t &out_width, size_t &out_height) {
+  switch (grid_size) {
+    case VectorGridSize::SIZE_1:
+      out_width = width;
+      out_height = height;
+      break;
+    case VectorGridSize::SIZE_2:
+      out_width = (width + 1) / 2;
+      out_height = (height + 1) / 2;
+      break;
+    case VectorGridSize::SIZE_4:
+      out_width = (width + 3) / 4;
+      out_height = (height + 3) /4;
+      break;
+    default:
+      out_width = width;
+      out_height = height;
+      break;
+  }
```

Comment:
Hmm... if VectorGridSize were just an `int`, as I insist it should, this would be reduced to:
```suggestion
  out_width = div_ceil(width, grid_size);
  out_height = div_ceil(height, grid_size);
```

Reply:
Done